.cicd/scripts/wrapper_srw_ftest.sh

```diff
@@ -34,6 +34,10 @@ if [[ "${SRW_PLATFORM}" == hera ]]; then
     fi
 fi
 
+if [[ "$TASK_DEPTH" = 0 ]] ; then
```

Comment:
Should a new variable be included that is set to `$TASK_DEPTH`, with a default of 9?  Unless `TASK_DEPTH` has been added to the environment variables already prepared on the Tier-1 platforms, I suspect that having an unbound variable will lead to failure of the `Functional Workflow Task Test` stage.

Reply:
yes. Agree. Need to set the default to '9'.

Reply:
For a Jenkins CI build, the Jenkinsfile defines the TASK_DEPTH variable in the Functional WorkflowTaskTests stage - lines 182 thru 185 - and the user input option is defined at the top of Jenkinsfile - line 20.
So I believe it should work okay during a CI build.

Using the wrapper_srw_ftest.sh manually would fail if TASK_DEPTH is not supplied, so this needs to be remedied.

Reply:
Are the contents of line 20 in the `Jenkinsfile` only set if the Jenkins tests are manually run in the Jenkins API?  Or, will it automatically set `SRW_WRAPPER_TASK_DEPTH` to be equal to 9 by default?

If it will automatically set `TASK_DEPTH`, then only a minor modification to allow for manual runs outside of Jenkins is necessary.  If the tests need to be manually run in the Jenkins API to set `TASK_DEPTH`, then the modification for manual runs outside of Jenkins will need to work with the automatic launch when the Jenkins label is added.

Reply:
Update Jenkinsfile to use default TASK_DEPTH=9

Reply:
The Jenkinsfile parameter SRW_WRAPPER_TASK_DEPTH will automatically be set to the first value in the list of possible values, in this case '9', back to the original default when using Jenkins.
When running the script manually, outside of Jenkins, the TASK_DEPTH parameter has always been optional, and when not supplied it defaults to '9'.